portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/Organizations.testcase

```diff
@@ -706,6 +706,67 @@ definition {
 		}
 	}
 
+	@description = "Use case for LPS-198960.Organization users and sub organizations are not mix across pages using interator button"
+	@priority = 4
+	test CanListedUsersanddSubOrganizationsAcrossPages {
+		task ("Given an Organization with two Users") {
+			JSONOrganization.addOrganization(organizationName = "Organization for Users");
+
+			for (var user : list "A,B") {
+				JSONUser.addUser(
+					userEmailAddress = "${user}@liferay.com",
+					userFirstName = "${user}_fn",
+					userLastName = "${user}_sn",
+					userScreenName = "${user}_sn");
+
+				JSONOrganization.assignUserToOrganization(
+					organizationName = "Organization for Users",
+					userEmailAddress = "${user}@liferay.com");
+			}
+		}
+
+		task ("And 4 Sub Organization within the same organization") {
+			for (var count : list "A,B,C,D") {
```

Comment:
Why counting on letters? It would be better to have Suborganization 1, 2, ... at this point

Reply:
I'm using the letters before names to be able to verify that the mix up doesn't happen, 
also taking into account the alphabetical order of the items